src/cargo/ops/cargo_new.rs

```diff
@@ -537,6 +543,7 @@ name = "{}"
 version = "0.1.0"
 authors = [{}]
 edition = {}
+publish = {}
```

Comment:
Could this be omitted when the default is crates.io?

Reply:
I have updated this to exclude when using default registry.